### ISSUE TITLE
Created a working Dockerfile to build on Raspberry ARM64

### ISSUE THREAD

**Issue by floohh** [dmunozv04/iSponsorBlockTV#142]
**Is your feature request related to a problem? Please describe.**
After some experimentation I was able to create a Dockerfile to get it to built on Raspberry with ARM64

**Describe the solution you'd like**
I've added gcc and std includes to the Dockerfile so aiohttp (modified the requirements to 3.9.3) can be successfully compiled and used.

**Describe alternatives you've considered**
I've played around with different versions of aiohttp but it did not work, needed to be compiled. 

**Additional context**
I am fairly new to GitHub if there is a better way to communicating this, I am sorry.

Here is the new Dockerfile:
[Dockerfile.txt](https://github.com/dmunozv04/iSponsorBlockTV/files/14453295/Dockerfile.txt)

**Comment by dmunozv04** [comment]
Hi there, we already have images for arm64 (but not for armv7). Does this new dockerfile allow it to build? Also, to reduce size; adding packages with apk; installing deps and removing build apk packages in one step would be ideal

**Comment by floohh** [comment]
Yes the Raspberry Pi Zero 2 W is arm7, and this new file allows it to build, but it takes a long time to compile aiohttp (10min on MacBook Pro). 

As this happens when pip is installing the requirements, those libraries can only be removed after that step. An updated Dockerfile is attached.

[Dockerfile2.txt](https://github.com/dmunozv04/iSponsorBlockTV/files/14460422/Dockerfile2.txt)

**Comment by dmunozv04** [comment]
Should be fixed now, there's a linux/arm/v7 image
